### PR TITLE
Update WordPress Authenticator to remove Lottie dependency

### PR DESCRIPTION
**Pull request by mokagio**
### Description

See https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/640.

### Testing instructions

As you can see, the UI tests are green and we do test the login flow extensively. On top of that, I used the Installable Build to test:

- Direct login via the iOS password manager prompt.
- Login via login button and 1Password autofill.
- Google login.
- That the signup flow works, i.e. results in me receiving a signup-link mail at the given address. 
- That the prologue carousel still works (unnecessary, given it's a [different class](https://github.com/woocommerce/woocommerce-ios/blob/595b3521748894649f63167cfd30ba8ba6d92fe7/WooCommerce/Classes/Authentication/Prologue/LoginPrologueViewController.swift) than the library's, but felt like it wouldn't hurt 😅).

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile
Comment (AliSoftware) [inline Podfile]: Of course, before merging, we'll want at least want https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/640 to be merged and shipped as a beta or stable.

I've added the `status: do not merge` label to the PR to make that more visible too and avoid accidental premature merge.

Comment (mokagio) [reply Podfile]: Nice. Better than just a comment 😅 Plus gets Peril to help us with it, too.

**Review by AliSoftware — APPROVED** [review]
:shipit:

**Pull request merged**
